File: setpaths/setpaths.m
function [libpath, datpath, resultpath, basepath] = setpaths
    fid = fopen('paths.txt');
    libpath = sscanf(strrep(fgetl(fid), ' ', ''), "libpath=%s");
    datpath = sscanf(strrep(fgetl(fid), ' ', ''), "datpath=%s");
    resultpath = sscanf(strrep(fgetl(fid), ' ', ''), "resultpath=%s");
    basepath = sscanf(strrep(fgetl(fid), ' ', ''), "basepath=%s");
end

File: setpaths/setpaths.py
from parse import *

def setpaths(paths_dir):
    with open('%s/paths.txt' % paths_dir, 'r') as f:
        all_lines = f.readlines()
    libpath = parse('libpath={}', "".join(all_lines[0].split()))
    datpath = parse('datpath={}', "".join(all_lines[1].split()))
    resultpath = parse('resultpath={}', "".join(all_lines[2].split()))
    basepath = parse('basepath={}', "".join(all_lines[3].split()))
    return libpath[0], datpath[0], resultpath[0], basepath[0]
